valid_palindrome.py

```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def isPalindrome(self, s: str) -> bool:
+        for i in s:
+            if not i.isalnum():
+                s = s.replace(i,"")
+        
+        s2 = s.lower()

```

Comment:
nice use of string methods, but just as a recommendation, look at how you can achieve the same without builtin functions (I do that just for curiosity)